charts/base-cluster/templates/backup/velero.yaml

```diff
@@ -77,6 +77,13 @@ spec:
             name: {{ get $existingSecret "name" | required (printf "You need to provide the existingSecret.name for %s" $name) }}
             key: {{ get $existingSecret "key" | default (printf "%s-%s" $providerName $name) }}
           {{- end }}
+          {{- if eq $providerName "minio" }}
+          {{- $providerSpec := get $spec.provider $providerName }}
+          config:
+            region: {{ ($providerSpec.region | default "Region1") | quote }}
+            s3ForcePathStyle: {{ ($providerSpec.forcePathStyle | default "true") | quote }}
```

Comment:
This is a boolean, are you sure this has to be quoted? Does it even work quoted? 🤔